utils/content.ts

```diff
@@ -221,15 +253,12 @@ export const getContentWithTag = (tag: string, contentType: IContentType) => {
         ...data,
         previewImage: data.previewImage || '/images/image-placeholder.png',
         id: uuid(),
-      };
+      } as IContent;
```

Comment:
Added type assertion because for TS the type returned had `previewImage` and `id` but this was wrong because in lines 253-255 we spread the `data` object and added `previewImage` and `id`

Reply:
hmm, let's try to avoid a type assertion if possible (for all type assertions)

usually it means that the types need to be more strict somewhere